### DISCUSSION TITLE
[Feature Request] Merchandise purchase restrictions

### DISCUSSION THREAD

**Discussion by Zhou-Shilin** [QuickShop-Community/QuickShop-Hikari#915] [Ideas]
_Google Translated_
Merchandise purchase/sale restrictions (limited purchase/sale)
In my server, I use the Quickshop system store (unlimited, buy) to issue currency. However, I soon found out that this is not appropriate because it causes inflation. So I wanted to limit the system store to, say, a total of 100 items per month. The limit here is not how many per player, it's how many of the item can be sold this month. 

Original Text / 原文

> 商品购买/售卖限制（限购/限卖）
> 在我的服务器中，我使用Quickshop系统商店（无限收购）来发行货币。然而，我很快发现这并不合适，因为它会导致通货膨胀。所以我想给系统商店限购，比如每月100件。这里的限制不是每个玩家限购多少，而是这个月一共能卖出多少件这种物品。

**Comment by Ghost-chu** [comment]
Why don't you just refill the shop inventory every month?
You can do that by yourself or by other 3rdparty plugins. Just found a way that able to put items into container.

> 你可以每个月都手动补充商店库存，或者直接找一个插件定时补充。总之随便什么能把物品塞进容器里就行。

**Reply by Zhou-Shilin** [reply]
This only solves the problem of _selling_ merchandise, it does not limit how many items the store can acquire in the way of regular restocking.

> 这只能解决卖商品的问题，不能以定期补货的方式限制商店能收购多少玩家的东西。
